pages/releases/4.3.html

```diff
@@ -0,0 +1,2337 @@
+						</div>
+					</div>
+				</div>
+
+				<div class="release-type release-contributors">
+					<div class="release-type-version release-contributors-version release-contributors-4_3">
+						<div class="release-type-item-bar release-contributors-item-bar">
+							<span class="release-type-item-title release-contributors-item-title">4.3</span>
+							<div
+								class="release-type-item-bar-bar release-contributors-item-bar-bar"
+								data-size="{{ page.4_3_contributors }}"
+								data-total="{{ page.4_3_contributors }}"
+							></div>
+							<span class="release-type-item-bar-number release-contributors-item-bar-number">0</span>
+							<span class="release-type-item-bar-desc release-contributors-item-bar-desc">Contributors</span>
+						</div>
+					</div>
+
+					<div class="release-type-version release-contributors-version release-contributors-4_2">
+						<div class="release-type-item-bar release-contributors-item-bar">
+							<span class="release-type-item-title release-contributors-item-title">4.2</span>
+							<div
+								class="release-type-item-bar-bar release-contributors-item-bar-bar"
+								data-size="{{ page.4_2_contributors }}"
+								data-total="{{ page.4_3_contributors }}"
+							></div>
+							<span class="release-type-item-bar-number release-contributors-item-bar-number">0</span>
+						</div>
+					</div>
+
+					<div class="release-type-version release-contributors-version release-contributors-4_1">
+						<div class="release-type-item-bar release-contributors-item-bar">
+							<span class="release-type-item-title release-contributors-item-title">4.1</span>
+							<div
+								class="release-type-item-bar-bar release-contributors-item-bar-bar"
+								data-size="{{ page.4_1_contributors }}"
+								data-total="{{ page.4_3_contributors }}"
+							></div>
+							<span class="release-type-item-bar-number release-contributors-item-bar-number">0</span>
+						</div>
+					</div>
+				</div>
+			</div>
+			<p class="desc">
+				From new nodes, to rendering improvements, to more stability, and
+				plenty of fixes. There's something for everyone.
+			</p>
+		</div> <!-- .release-header-container -->
+	</header>
+
+	<main class="release-main">
+		<div class="release-main-container container">
+			<!-- Presentation -->
+			<div id="presentation" class="release-section release-section--presentation">
+				<godot-release-box
+					type="bottom"
+				>
+					<div slot="blockquote">
+						Here is a video from one of our content-creators!
+					</div>
+					<div slot="media">
+						<iframe style="width: 100%; aspect-ratio: 16/9; height: min-content;" width="560" height="315" src="https://www.youtube.com/embed/bbatLr98fEY?si=LvzwlTzH9ZLRJzRK" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Comment:
I assume this youtube video is a placeholder? If so, what do you intend to put here?

Reply:
Oops, yes, it's a placeholder. We'll try to put a video from a content-creator there that is explaining the update on a video form. For an example, [see the Blender 4.1 page](https://www.blender.org/download/releases/4-1/).